src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/ErrorMessage.java

```diff
@@ -20,7 +20,7 @@
 
 public class ErrorMessage {
 
-    private Exception exception;
+    protected Exception exception;
```

Comment:
Could you remove this field and pass it to `fetchReason` and `fetchDetails` instead?